lib/jira/config.js

```diff
@@ -1,12 +1,12 @@
 process.env.NODE_TLS_REJECT_UNAUTHORIZED = 0;
-
 const iconPath = process.cwd() + '/resources/icons/';
 const cfgPath = process.env.HOME + '/.alfred-jira/';
 module.exports = {
   cfgPath: cfgPath,
   cfgFile: 'config.json',
   cacheFile: 'cache.json',
   iconPath: iconPath,
+  sepperator: process.env.sepperator || '► ',
```

Comment:
Any reason to spell it `sepperator ` instead of `separator`? If it’s simply a misspelling, it’d be great to spell it correctly!